_posts/wasmer-go-embedding-1.0.md

```diff
@@ -0,0 +1,534 @@
+---
+title: 'Wasmer Go embedding 1.0 liftoff'
+excerpt: 'Announcing the immediate availability of the Wasmer Go embedding 1.0 version!'
+date: '2021-02-22T15:35:07.322Z'
+author: Ivan Enderlin
+---
+
+We are delighted to announce the availability of the [Wasmer Go
+embedding 1.0 version](https://github.com/wasmerio/wasmer-go).
+
+About 1.5 years ago we first release `wasmer-go`, the Wasmer embedding
+for Go. The reception by the community was beyond our expectations,
+the repercussion of launching [the fastest WebAssembly runtime for
+Go](https://medium.com/wasmer/announcing-the-fastest-webassembly-runtime-for-go-wasmer-19832d77c050)
+was a great way to start! A community of more than 1'300 people gives
```

Comment:
Number separator is a mess. In Europe Latin countries in general (except English, more or less), if you use a comma, it's for decimal separation, so 4,2 is 4.2 in English.

I'm used to use a ' for the grouping separator so that it makes everything unambiguous. But it's not what Wikipedia recommends: https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Grouping_of_digits:

> Numbers with exactly four digits left of the decimal point may optionally be grouped (either 1,250 or 1250), with consistency within any given article.

OK, let's use 1,300 then :-)!